### ISSUE TITLE
//@stylint ignore is acting like //@stylint on

### ISSUE THREAD

**Issue by camjc** [rossPatton/stylint#185]
```stylus
.foo
  margin 0 !important // @stylint ignore
  padding 0 !important
```

The ignore stops the padding's important being found.
Looks like it happens to the bottom of the file. I'm running 1.1.1

**Comment by robmunro** [comment]
Having the same problem.

**Comment by rossPatton** [comment]
should be fixed in 1.1.3

**Comment by camjc** [comment]
You're on the ball! Thanks so much mate.
